.github/workflows/test-docs.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
 jobs:
   # Test building the docs (html, linkcheck, doctest, coverage)
   test-docs:
-    runs-on: ubuntu-latest-4-cores
+    runs-on: macos-latest
```

Comment:
I think this change is going to need to be reverted. It is causing the build to slow down significantly.

Reply:
I reverted this change.